asf-site-src/pom.xml

```diff
@@ -84,7 +84,6 @@
             </goals>
             <configuration>
               <executable>${rdm.bin}</executable>
-              <requiresOnline>true</requiresOnline>
```

Comment:
oh why were these here in the first place?

Reply:
At some point in time, we either used a plugin that was offline/online aware or they removed the functionality from maven.  But they were there so that you could tell maven you were offline and it wouldn't try to connect to JIRA to generate the release notes, etc.